libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -584,18 +583,6 @@ internal class MapboxTripSession(
         action: (BannerInstructions) -> Unit
     ) {
         ifNonNull(bannerInstructionEvent.bannerInstructions) { bannerInstructions ->
-            val bannerView = bannerInstructions.view()
-            val bannerComponents = bannerView?.components()
-            ifNonNull(bannerComponents) { components ->
-                components.forEachIndexed { index, component ->
-                    component.takeIf { it.type() == BannerComponents.GUIDANCE_VIEW }?.let { c ->
-                        components[index] =
-                            c.toBuilder()
-                                .imageUrl(c.imageUrl()?.plus("&access_token=$accessToken"))
-                                .build()
-                    }
-                }
-            }
```

Comment:
I have removed this because `"&access_token=$accessToken"` is different for junctions whereas different for signboards. Also, this won't work for us because `accessToken` passed into `MapboxTripSession` is the one that renders Maps, but we need specific route tokens to render Zenrin based junctions and HERE based signboards.

Reply:
Aren't the `accessToken`s going to be eventually the same? How are we going to handle separate `accessToken`s for Maps and Directions in the long term? 🤔

Reply:
Even more if Signboard and Junction `accessToken`s are different...

Reply:
>Aren't the accessTokens going to be eventually the same?

They should from customer perspective, but while development we have different tokens for all and hence it doesn't work. We could possibly add some field in `NavigationOptions` that allows to set `routeAccessToken`.
cc @Guardiola31337